runtime/common/src/lib.rs

```diff
@@ -160,121 +161,27 @@ pub mod parachains {
 		};
 	}
 
-	pub mod polkadot {
-		pub mod asset_hub {
-			pub const PARA_ID: u32 = 1000;
-			pub const ASSET_PALLET_INDEX: u8 = 50;
-
-			parachain_asset_location!(USDC, 1337);
-			parachain_asset_location!(USDT, 1984);
-			parachain_asset_location!(PINK, 23);
-		}
-
-		pub mod equilibrium {
-			pub const PARA_ID: u32 = 2011;
-			pub const ASSET_PALLET_INDEX: u8 = 11;
-
-			parachain_asset_location!(EQ, 25_969);
-			parachain_asset_location!(EQD, 6_648_164);
-		}
-
-		pub mod moonbeam {
-			use xcm::latest::{
-				Junction::{AccountKey20, PalletInstance, Parachain},
-				Junctions::{X2, X3},
-			};
-
-			pub const PARA_ID: u32 = 2004;
-			pub const ASSET_PALLET_INDEX: u8 = 110;
-			pub const BALANCES_PALLET_INDEX: u8 = 10;
-
-			// The address of the BRZ token on Moonbeam `0x3225edCe8aD30Ae282e62fa32e7418E4b9cf197b` as byte array
-			pub const BRZ_ASSET_ACCOUNT_IN_BYTES: [u8; 20] = [
-				50, 37, 237, 206, 138, 211, 10, 226, 130, 230, 47, 163, 46, 116, 24, 228, 185, 207,
-				25, 123,
-			];
-
-			parachain_asset_location!(
-				BRZ,
-				X3(
-					Parachain(PARA_ID),
-					PalletInstance(ASSET_PALLET_INDEX),
-					AccountKey20 { network: None, key: BRZ_ASSET_ACCOUNT_IN_BYTES }
-				)
-			);
-
-			parachain_asset_location!(
-				GLMR,
-				X2(Parachain(PARA_ID), PalletInstance(BALANCES_PALLET_INDEX))
-			);
-		}
-
-		pub mod polkadex {
-			use xcm::latest::{Junction::Parachain, Junctions::X1};
-
-			pub const PARA_ID: u32 = 2040;
-
-			parachain_asset_location!(PDEX, X1(Parachain(PARA_ID)));
-		}
-	}
-
-	pub mod kusama {
+	// These values are shared accross both Pendulum and Kusama
+	pub mod asset_hub {
```

Comment:
I think we have to define this relay-chain specific. If you compare the assets on [Kusama's](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fksm-rpc.stakeworld.io%2Fassethub#/assets) AssetHub to [Polkadot's](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fstatemint.api.onfinality.io%2Fpublic-ws#/assets) Assethub you'll see that the assets are different.